packages/E2ETest/windows/ReactUWPTestApp/Assets/TreeDump/ControlStyleRegularBorder.txt

```diff
@@ -22,7 +22,7 @@
       HorizontalAlignment=Left
       Margin=0,0,0,0
       Padding=0,0,0,0
-      RenderSize=51,50
+      RenderSize=51.33333,50
```

Comment:
>51.33333 [](start = 17, length = 8)

The baseline change look suspicious. I doubt that changing the tools chain could cause it.

Reply:
Yeah, I'm not sure what's going on here. I'm going to revert the changes to the masters and go through the CI logs again.